datalad/distribution/create.py

```diff
@@ -285,11 +284,10 @@ def __call__(
                                    relativepath=subdsrelpath)
             # this will have staged the changes in the superdataset already
             if save:
-                Save.__call__(
-                    message='[DATALAD] added subdataset',
-                    dataset=dataset,
-                    auto_add_changes=False,
-                    recursive=False)
+                save_dataset(
```

Comment:
shouldn't it just be `dataset.save([tbds.path, ...` ?  would be a confusing API to have both